<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDispOptionsPropSheet</class>
 <widget class="QDialog" name="CDispOptionsPropSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Options</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>350</width>
     <height>342</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_25">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="dispOptsCardBacksPage">
       <attribute name="title">
        <string>Card Backs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose card back&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_27">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_26">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QToolButton" name="cardBack0">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback1.png</normaloff>:/resources/cardback1.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="cardBack1">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback22.png</normaloff>:/resources/cardback22.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="cardBack2">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback3.png</normaloff>:/resources/cardback3.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QToolButton" name="cardBack3">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback5.png</normaloff>:/resources/cardback5.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="cardBack4">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback11.png</normaloff>:/resources/cardback11.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="cardBack5">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/resources/cardback2.png</normaloff>:/resources/cardback2.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>80</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dispOptsMiscPage">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>231</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="backgroundColor">
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose background color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Background Color...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colorWatch">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDispOptionsPropSheet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDispOptionsPropSheet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
